sdk/core/Azure.Core/src/Pipeline/LoggingPolicy.cs

```diff
@@ -90,79 +72,32 @@ private async ValueTask ProcessAsync(HttpPipelineMessage message, ReadOnlyMemory
                 ProcessNext(message, pipeline);
             }
 
-
-
             var after = Stopwatch.GetTimestamp();
 
             bool isError = message.ResponseClassifier.IsErrorResponse(message);
 
-            var textResponse = ContentTypeUtilities.TryGetTextEncoding(message.Response.Headers.ContentType, out Encoding? responseTextEncoding);
-
-            bool logResponseContent = _logContent && message.Response.ContentStream != null;
-            bool wrapResponseContent = logResponseContent && message.Response.ContentStream?.CanSeek == false && s_eventSource.ShouldLogContent(isError);
+            ContentTypeUtilities.TryGetTextEncoding(message.Response.Headers.ContentType, out Encoding? responseTextEncoding);
 
-            if (wrapResponseContent)
-            {
-                message.Response.ContentStream = new LoggingStream(message.Response.ClientRequestId, s_eventSource, message.Response.ContentStream!, isError, responseTextEncoding);
-            }
+            bool wrapResponseContent = message.Response.ContentStream != null &&
+                                       message.Response.ContentStream?.CanSeek == false &&
```

Comment:
nit: Is the null conditional needed here since we are already checking for not null?